docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -1615,27 +1634,27 @@ Default stratifications to all observers should include scenario and input draw.
     - Default
     - Default
   * - 20.0
+    - :ref:`In-hospital (CEmONC) delivery estimates from HS team <facility_choice_choosing_bemonc_cemonc_section>`. See `PR with diff here <https://github.com/ihmeuw/vivarium_research/pull/1792/>`__
+    - Baseline 
+    - ``model20.0``
+    - Default
+    - Default
+    - Default
+  * - 21.0
     - Larger run for neonatal mortality V&V with "neonatal all-cause mortality risk", "neonatal cause-specific mortality risks", and "impossible neonatal CSMRisk" observers.
     - Baseline
-    - ``model20.0``
+    - ``model21.0``
     - For this run only, 10,000,000 population size per draw
     - Default
     - Default, note addition of "neonatal all-cause mortality risk", "neonatal cause-specific mortality risks", and "impossible neonatal CSMRisk" observers.
-  * - 21.0* (can be moved to before GBD 2023 update/19.0 if needed following an update of the cause IDs to be compatible with GBD 2023)
+  * - 22.0*
     - Inclusion of the :ref:`residual maternal disorders <2021_cause_residual_maternal_disorders_mncnh>` and :ref:`abortion/miscarriage/ectopic pregnancy maternal disorders <2021_cause_abortion_miscarriage_ectopic_pregnancy_causes_mncnh>` cause models
     - Baseline
-    - ``model21.0``
-    - Default
-    - Default
-    - Default
-  * - 22.0 (can be moved to before GBD 2023 update/19.0 if needed)
-    - :ref:`In-hospital (CEmONC) delivery estimates from HS team <facility_choice_choosing_bemonc_cemonc_section>`. See `PR with diff here <https://github.com/ihmeuw/vivarium_research/pull/1792/>`__
-    - Baseline 
     - ``model22.0``
     - Default
     - Default
     - Default
-  * - 23.0 (can be moved to before GBD 2023 update/19.0 if needed)
+  * - 23.0*
```

Comment:
This model doesn't have any research blocks so I think we can remove the asterisk but I don't think we should start on any of it until we fix the existing hemoglobin bugs/do the refactor